### PR TITLE
[chart.js] Fix dafaults and controllers Chart

### PR DESCRIPTION
**Pull request by guillaume-ro-fr**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [release](https://github.com/chartjs/Chart.js/releases/tag/v2.7.1)
- [ ] Increase the version number in the header if appropriate. **--> Patch**
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

With this correction, we can extend type with new ones.

### CODE REVIEW

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -36,6 +37,11 @@ declare class Chart {
 
     static defaults: {
         global: Chart.ChartOptions & Chart.ChartFontOptions;
+        [key: string]: any;
```
Comment (andy-ms) [inline types/chart.js/index.d.ts]: Could you explain in more detail why this change is needed?

**Review by ghost — CHANGES_REQUESTED** [review]

File: types/chart.js/index.d.ts (hunk 1 above)
Comment (guillaume-ro-fr) [reply types/chart.js/index.d.ts]: If you want to extend an actual chart type on `chart.js`, you also need to create a default value for this chart, because of a bug. See [this doc](http://www.chartjs.org/docs/latest/developers/charts.html#extending-existing-chart-types) : 

> // Sets the default config for 'derivedBubble' to be the same as the bubble defaults. 
> // We look for the defaults by doing Chart.defaults[chartType]
> // It looks like a bug exists when the defaults don't exist
> Chart.defaults.derivedBubble = Chart.defaults.bubble;

Comment (andy-ms) [reply types/chart.js/index.d.ts]: OK, thanks!
